radareorg/cutter
# Add missing debug dialogs

**Detailed description**

- Added debugTaskDialog to all debug actions that the user can use to break the process(Not all debug plugins support/should support breaking open()) or at least see the progress instead of freezing.
- Added a native debug dialog to pass command line arguments to `ood`
- Added a warning dialog when entering debug
- Fixed a race condition in asyncCmd

**Test plan (required)**

- See that the debug warning pops up only once per session
![debugdialog](https://user-images.githubusercontent.com/5659696/70470751-a48efa80-1ac3-11ea-8b07-41ce08b359e9.PNG)
- Check if debugTaskDialog is created after starting native debug, remote debug or attaching
![debugtaskdialog](https://user-images.githubusercontent.com/5659696/70470752-a48efa80-1ac3-11ea-951a-6e68946d52a5.PNG)
- Print argv after entering params in the native debug dialog
![ood args](https://user-images.githubusercontent.com/5659696/70470750-a3f66400-1ac3-11ea-858f-042309de6dff.PNG)

**Related issues**

Depends on the ProcessesWidget PR #1894
